docs/source/examples/input_schemas.rst

```diff
@@ -0,0 +1,119 @@
+Action Provider Input Schemas
+=============================
+
+We support user-written JSONSchema and Pydantic_ based schema defintions. In
```

Comment:
In this one I tried out a different way of making rst links - I added `.. _Pydantic: https://pydantic-docs.helpmanual.io/` to the bottom of the doc so that hopefully the link appears once it renders.